### PR TITLE
Chapter 8: Basic login

### PR DESCRIPTION
**Pull request by ManhLeDuc**
Chapter 8: Basic login
Link báo cáo: https://docs.google.com/document/d/1xTmvZTO-CKhf_0kGjr8bgi_68V1vvGJDqZbvZoH2wbE/edit?usp=sharing
image

### CODE REVIEW

**Comment by ManhLeDuc** [comment]
ready

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+    if user&.authenticate params[:session][:password]
+      log_in user
+      redirect_to controller: "users", action: "show", id: user.id
```
Comment (ngocvt-0484) [inline app/controllers/sessions_controller.rb]: redirect_to users_path(user)

dc k?

**Comment by ManhLeDuc** [comment]
ready

File: app/assets/stylesheets/sessions.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Sessions controller here.
+// They will automatically be included in application.css.
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/sessions.scss]: remove file

File: app/views/users/new.html.erb [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <h1><%= t ".name" %></h1>
 <div class="row">
   <div class="col-md-6 col-md-offset-3">
-    <%= form_with(model: @user, local: true) do |f| %>
+    <%= form_with model: @user, local: true do |f| %>
```
Comment (ngocvt-0484) [inline app/views/users/new.html.erb]: dùng form_for dc k

**Comment by ManhLeDuc** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**